[2024-01-24T10:05:20.371+0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_comments_score.train_model scheduled__2024-01-24T01:05:00+00:00 [queued]>
[2024-01-24T10:05:20.384+0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_comments_score.train_model scheduled__2024-01-24T01:05:00+00:00 [queued]>
[2024-01-24T10:05:20.385+0800] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-24T10:05:20.385+0800] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-24T10:05:20.385+0800] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-24T10:05:20.403+0800] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): train_model> on 2024-01-24 01:05:00+00:00
[2024-01-24T10:05:20.409+0800] {standard_task_runner.py:55} INFO - Started process 601501 to run task
[2024-01-24T10:05:20.416+0800] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'youtube_comments_score', 'train_model', 'scheduled__2024-01-24T01:05:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/youtube_comments_score.py', '--cfg-path', '/tmp/tmpjz5yaa_i']
[2024-01-24T10:05:20.418+0800] {standard_task_runner.py:83} INFO - Job 80: Subtask train_model
[2024-01-24T10:05:20.553+0800] {task_command.py:389} INFO - Running <TaskInstance: youtube_comments_score.train_model scheduled__2024-01-24T01:05:00+00:00 [running]> on host ml
[2024-01-24T10:05:20.675+0800] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=youtube_comments_score
AIRFLOW_CTX_TASK_ID=train_model
AIRFLOW_CTX_EXECUTION_DATE=2024-01-24T01:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-24T01:05:00+00:00
[2024-01-24T10:05:20.677+0800] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-24T10:05:20.678+0800] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/am/project/scripts/train_model.py']
[2024-01-24T10:05:20.692+0800] {subprocess.py:86} INFO - Output:
[2024-01-24T10:05:21.546+0800] {subprocess.py:93} INFO - /home/am/project/scripts/train_model.py:3: DeprecationWarning:
[2024-01-24T10:05:21.546+0800] {subprocess.py:93} INFO - Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
[2024-01-24T10:05:21.546+0800] {subprocess.py:93} INFO - (to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
[2024-01-24T10:05:21.546+0800] {subprocess.py:93} INFO - but was not found to be installed on your system.
[2024-01-24T10:05:21.546+0800] {subprocess.py:93} INFO - If this would cause problems for you,
[2024-01-24T10:05:21.546+0800] {subprocess.py:93} INFO - please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
[2024-01-24T10:05:21.546+0800] {subprocess.py:93} INFO - 
[2024-01-24T10:05:21.547+0800] {subprocess.py:93} INFO -   import pandas as pd
[2024-01-24T10:05:21.553+0800] {subprocess.py:93} INFO - Traceback (most recent call last):
[2024-01-24T10:05:21.553+0800] {subprocess.py:93} INFO -   File "/home/am/project/scripts/train_model.py", line 11, in <module>
[2024-01-24T10:05:21.553+0800] {subprocess.py:93} INFO -     with open('/home/am/project/models/data.pickle', 'wb') as f:
[2024-01-24T10:05:21.554+0800] {subprocess.py:93} INFO - FileNotFoundError: [Errno 2] No such file or directory: '/home/am/project/models/data.pickle'
[2024-01-24T10:05:21.740+0800] {subprocess.py:97} INFO - Command exited with return code 1
[2024-01-24T10:05:21.764+0800] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/am/project/venv/lib/python3.10/site-packages/airflow/operators/bash.py", line 196, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-01-24T10:05:21.768+0800] {taskinstance.py:1327} INFO - Marking task as UP_FOR_RETRY. dag_id=youtube_comments_score, task_id=train_model, execution_date=20240124T010500, start_date=20240124T020520, end_date=20240124T020521
[2024-01-24T10:05:21.788+0800] {standard_task_runner.py:100} ERROR - Failed to execute job 80 for task train_model (Bash command failed. The command returned a non-zero exit code 1.; 601501)
[2024-01-24T10:05:21.831+0800] {local_task_job.py:212} INFO - Task exited with return code 1
[2024-01-24T10:05:21.869+0800] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
